docs/software-design-federation-gateway-service.md

```diff
@@ -306,13 +308,35 @@ A DiagnosisKeyBatch can contain more than one DiagnosisKey. To make sure that th
 verifier (Federation Gateway) process the same byte stream, the DiagnosisKey objects in the DiagnosisKeyBatch must be 
 sorted by KeyData (see method **sortBatchByKeyData** in [BatchSignatureUtils.java](https://github.com/eu-federation-gateway-service/efgs-federation-gateway/blob/master/src/main/java/app/coronawarn/interop/federationgateway/utils/BatchSignatureUtils.java)).   
 
-### 3.3. Certificate Verification
+### 3.3. Certificate Verification during OnBoarding
```

Comment:
Thanks for your input. Splitting the Signing-Cert-Verification in two chapters makes sense for me. But the onboarding process will not be done by EFGS Gateway. It is not planned to have such an endpoint. The onboarding process wil be done manually.
It would be great if you will adopt wording to this.